src/main/webapp/new_post.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>New Post</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    </head>
+
+
+    <body>
+        <h1>Create new Post</h1>
+        <hr>
+        <form>
```

Comment:
Remember to add the method attribute and the action since it is a form.

Reply:
I didn't add the method yet because I have not created the servlet yet